src/pages/Regions/Region.tsx

```diff
@@ -202,13 +213,49 @@ const Region = () => {
     });
   }
 
+  /*const handleSubmit = async (values: RegionFollower) => {
+  *  const newCity: CityProfile = {
+  *    cityURL: values.cityURL,
+  *    description: values.cityDescription,
+  *    email: values.email,
+  *    head: new CityAdmin(),
+  *    headDeputy: city.headDeputy,
+  *    houseNumber: values.houseNumber,
+  *    id: city.id,
+  *    logo: city.logo?.length === 0 ? null : city.logo,
+  *    officeNumber: values.officeNumber,
+  *    name: values.name,
+  *    phoneNumber: values.phoneNumber,
+  *    postIndex: values.postIndex,
+  *    region: values.region,
+  *    street: values.street,
+  };*/
+
+
+  /*function seeAddFollowerModal(follower: RegionFollower) {
+  *  return Modal.confirm({
+  *    title: `Ви впевнені, що хочете підтвердити заявку на створення станиці ${follower.cityName}?`,
+  *    icon: <ExclamationCircleOutlined />,
+  *    okText: "Так, створити",
+  *    okType: "danger",
+  *    cancelText: "Скасувати",
+  *    maskClosable: true,
+  *    onOk() {
+  *      {
+  *        //await handleSubmit(follower);
+  *      }
+  *    },
+  *  });
+  }*/
+
   const getRegion = async () => {
     setLoading(true);
 
     try {
       const response = await getRegionById(id);
       const response1 = await getRegionAdministration(id);
       const responce2 = await cityNameOfApprovedMember(userApi.getActiveUserId());
+      const response3 = await getRegionFollowers(id);
```

Comment:
Please, provide an appropriate name
This name doesn't meet with conventions

Reply:
Will fix it with the next commit